CHANGELOG.md

```diff
@@ -1,5 +1,109 @@
 ### [Versions](https://mui.com/versions/)
 
+## 5.1.0
+
+<!-- generated comparing v5.0.6..master -->
+
+_Nov 8, 2021_
+
+A big thanks to the 31 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎉 Support custom elements under `ButtonGroup`
+- 🛠 Add support for arrays in the `sx` prop
```

Comment:
What does this mean?

Reply:
https://github.com/mui-org/material-ui/pull/29297

Reply:
Will link the PRs on both